tests/test_apps/export-benchmark/src/exportbenchmark/ExportBenchmark.java

```
@@ -0,0 +1,154 @@
+ */
+
+package exportbenchmark;
+
+import org.voltdb.VoltTable;
+import org.voltdb.client.Client;
+import org.voltdb.client.ClientConfig;
+import org.voltdb.client.ClientFactory;
+import org.voltdb.client.ClientResponse;
+import org.voltdb.client.VoltBulkLoader.BulkLoaderFailureCallBack;
+import org.voltdb.client.VoltBulkLoader.VoltBulkLoader;
+
+public class ExportBenchmark {
+    
+    private Client client;
+    
+    long count = 10000;
+    String host = "localhost";
+    int port = 21212;
+    
+    /**
+     * Creates a new instance of the test to be run.
+     * Establishes a client connection to a voltdb server, which should already be running
+     * @param args The arguments passed to the program
+     */
+    public ExportBenchmark(String[] args) {
+        parseCommandLine(args);
+        
+        ClientConfig clientConfig = new ClientConfig();
+        clientConfig.setReconnectOnConnectionLoss(true);
+        clientConfig.setClientAffinity(true);
+        client = ClientFactory.createClient(clientConfig);
+    }
+    
+    /**
+     * Loops through the command line arguments & applies them
+     * @param args The arguments passed to the program
+     */
+    private void parseCommandLine(String[] args) {
+        for (int i = 0; i < args.length; i++) {
+            String[] arg = args[i].split("=");
+            if (arg[0].equals("--count")) {
+                try {
+                    count = Long.parseLong(arg[1]);
+                } catch (NumberFormatException e) {
+                    System.err.println("'" + arg[1] + "': not a valid number");
+                    System.exit(1);
+                }
+            } else if (arg[0].equals("--servers")) {
+                host = arg[1];
+            } else if (arg[0].equals("--port")) {
+                port = Integer.parseInt(arg[1]);
+            } else {
+                System.err.println("Unknown argument: '" + args[i] + "' - ignoring");
+            }
```

should fail instead of ignoring